packages/lib-sourcify/test/SourcifyChain.spec.ts

```diff
@@ -286,8 +286,16 @@ describe('SourcifyChain', () => {
 
   describe('Unit tests', () => {
     let hardhatNodeProcess: ChildProcess;
+    let sourcifyChain: SourcifyChain;
```

Comment:
Can we move this describe block out to the topmost level? I don't like the shadowing of the `sourcifyChain` variable.